src/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,20 @@
+import react, { createElement } from "react";
+
+const geners = ['Action', 'Comedy', 'Drama', 'Fantasy'];
+
+class GenreSelect extends react.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return createElement(
```

Comment:
createElement API needed only for Counter.js file, in general we use JSX

Reply:
fixed